### PR TITLE
add Overture Tools

### PR DESCRIPTION
**Pull request by ysake**
Add a new Cask.
- Overture Tools v.2.3.6

### CODE REVIEW

**Comment by vitorgalvao** [comment]
There was a PR template when you submitted, it must be filled.

File: Casks/overture.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'overture' do
+  version '2.3.6'
+  sha256 '623355be84033257f3434d2f85a2a632f28f4adbeee4a897b92be53e147846a4'
+
+  url 'https://github.com/overturetool/overture/releases/download/Release%2F2.3.6/Overture-2.3.6-macosx.cocoa.x86_64.zip'
```
Comment (vitorgalvao) [inline Casks/overture.rb]: `url "https://github.com/overturetool/overture/releases/download/Release%2F#{version}/Overture-#{version}-macosx.cocoa.x86_64.zip"`

Comment (vitorgalvao) [inline Casks/overture.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

Comment (vitorgalvao) [inline Casks/overture.rb]: Needs a `# github.com/overturetool/overture was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

**Comment by ysake** [comment]
Thank you for the comment .
I fix them .

**Comment by vitorgalvao** [comment]
You all need to add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Comment by ysake** [comment]
sorry.. I added.

**Pull request merged**